README.md

```diff
@@ -13,20 +13,20 @@ integrations.
 - Define your data and endpoints with OpenAPI
 - No maintenance of generated code
 
-## Status: Developer Preview #3
+## Status: General Availability
 
-LoopBack 4 is a work in progress, the public API is frequently changed in
-backward incompatible ways. See
-[Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
-for more details.
+LoopBack 4 has GA'ed. 
```

Comment:
Using @bajtos' suggestion on the v4.loopback.io PR, perhaps we can use the same:

LoopBack 4 GA (General Availability) has been released in October 2018, read more in [the announcement post](http://strongloop.com/strongblog/loopback-4-ga). 

Thanks!